src/cephadm/cephadm

```diff
@@ -4600,7 +4725,15 @@ def _get_parser():
     parser_bootstrap.add_argument(
         '--initial-dashboard-password',
         help='Initial password for the initial dashboard user')
-
+    parser_bootstrap.add_argument(
+        '--dashboard-ssl-disabled',
+        action='store_true',
+        help='Disable secured connections to the dashboard')
```

Comment:
What do you think of keeping ssl unconditionally enabled? would that work for you? or do you have a specific use case?

Reply:
I just try to keep intact the dashboard functionality. 
When enabling the dashboard, the user can choose to use or not SSL to access dashboard. Why do we need  to remove this possibility? why do we need to restrict final user freedom?

Reply:
by being opinionated, we can reduce the complexity of cephadm. we have to think carefully where we want to invest our time in. Users can always disable SSL afterwards, if they want to.